routes/employee.js

```
@@ -153,9 +195,33 @@ router.get('/:id', auth, (req, res) => {
+        ),
+        lastModified:
+          employee.metadata?.lastModified || new Date().toISOString(),
+        profileCompleteness:
+          employee.personalInfo && employee.jobDetails ? 'complete' : 'partial',
+        accessLevel: 'standard',
+      };
+    }
+
+    // NEW: Filter sensitive information based on access level
+    if (req.query.publicView === 'true') {
+      delete responseData.salary;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)